go/billing/forms.py

```diff
@@ -91,22 +92,8 @@ def __init__(self, *args, **kwargs):
         self.fields['account_number'].widget = forms.HiddenInput()
 
     def load_credits(self):
-        account = self.instance
-
-        # Create a new transaction
-        transaction = Transaction.objects.create(
-            account_number=account.account_number,
-            credit_amount=self.cleaned_data['credit_amount'])
-
-        # Update the selected account's credit balance
-        account.credit_balance += transaction.credit_amount
-        account.last_topup_balance = account.credit_balance
-
-        account.save()
-
-        # Update the transaction's status to Completed
-        transaction.status = Transaction.STATUS_COMPLETED
-        transaction.save()
+        return load_account_credits(
```

Comment:
``load_account_credits`` doesn't return anything so wondering whether we should not `return` here.